README.md

```diff
@@ -11,33 +11,6 @@ You will require the SDK on your classpath, so go ahead and add it to your gradl
 implementation("io.getunleash:unleash-android-proxy-sdk:${unleash.sdk.version}")
```

Comment:
Maybe add a note in the README about which Android API version this SDK supports?

Reply:
Will do